client/mysqldump.c

```diff
@@ -6123,6 +6135,8 @@ static int do_stop_slave_sql(MYSQL *mysql_con)
           mysql_free_result(slave);
           return 1;
         }
+        strmov(stopped_replicas[n_stopped_replicas++],
+               multi_source ? row[0] : "");
```

Comment:
pedantically supporting no-multi-source on a best effort basis
(But those pre-10.0 versions were long-EOL… 😕)
https://github.com/MariaDB/server/blob/6aa47fae304bd1f3efc0ebd2035bc1481ded73a3/client/mysqldump.c#L7010-L7015

Reply:
Oh? 🧐 Our [Engineering Policy](https://mariadb.com/engineering-policies/) says that 10.5 *Enterprise* goes EOL after 16th this month.
It should be safe to refactor away no-multi-source support.